README.md

```diff
@@ -5,10 +5,15 @@ _See [the dashboard](https://mozilla.github.io/standards-positions/) for Mozilla
 ## What is this Repository?
 
 This repo is where [Mozilla](https://mozilla.org/) decides and documents what it thinks about
-emerging technical specifications for the Web. Typically they're draft documents in the
-[IETF](https://ietf.org/), [W3C](https://w3.org/) (including the [WICG](https://wicg.github.io/)),
-[WHATWG](https://whatwg.org/), and [Ecma TC39](https://github.com/tc39), but they could come from
-elsewhere too.
+emerging technical specifications for the Web. Typically such specs are standards-track drafts in the
+[IETF](https://ietf.org/), [W3C](https://w3.org/),
+[WHATWG](https://whatwg.org/), and [Ecma TC39](https://github.com/tc39).
+We may also provide positions on documents located outside a standards process, 
+e.g. those in W3C Community Groups such as [WICG](https://wicg.github.io/) 
+or other community incubation efforts with well-understood
+<abbr title="intellectual property rights">IPR</abbr> terms, 
+but as a general rule believe that any such work must be moved to a formal standards track 
+before it’s appropriate for adoption as part of the web platform.
```

Comment:
This text might be better moved to the CONTRIBUTING.md file where we have a lot more detail on how to ask for a position and what we require of people who are asking.  This is a lot of detail to put up front.